pkg/features/embedded/grub-default-bootargs/etc/elemental/bootargs.cfg

```diff
@@ -1,9 +1,12 @@
 # bootargs.cfg inherits from grub.cfg several context variables:
 #   'img' => defines the image path to boot from. Active img is statically defined, does not require a value
+#   'mode' => active/passive/recovery, mode to boot
 #   'state_label' => label of the state partition filesystem
 #   'oem_label' => label of the oem partition filesystem
 #   'recovery_label' => label of the recovery partition filesystem
 #   'snapshotter' => snapshotter type, assumes loopdevice type if undefined
+#   'snap_arg' => kernel args for snapshotter
```

Comment:
nit: this is not inherited, this is not part of an _API_ this is just a local variable. I'd not list it here.

Reply:
Yep, removed in latest commit!